internal/pkg/plugin/manager.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config/v1"
```

Comment:
Ditto moving internal import to bottom of import list.
Ditto on naming this import config_v1